proposals/839-outreach-committee/kubeflow-outreach-committee.md

```diff
@@ -0,0 +1,31 @@
+# KEP- 839: Proposal for the Creation of the Kubeflow Outreach Committee (KOC)
+
+## Overview
+
+We propose the creation of the **Kubeflow Outreach Committee (KOC)** to lead efforts in growing and engaging the broader AI/ML community, not just the Kubeflow community. This committee will be responsible for fostering visibility, engagement, and inclusivity, as well as helping to expand Kubeflow’s presence and user base.
+
+## Objectives
+
+- Lead efforts to grow and engage the AI/ML community through events, educational materials, and advocacy for Kubeflow.
+- Onboard new contributors and users to Kubeflow, particularly from diverse and underrepresented communities.
```

Comment:
I think instead of onboard should be focus on growing communities from diverse and underrepresented communities.